cmd/neofs-adm/internal/modules/morph/n3client.go

```diff
@@ -20,12 +20,14 @@ type clientContext struct {
 }
 
 func getN3Client(v *viper.Viper) (*client.Client, error) {
+	const maxConnsPerHost = 1
```

Comment:
Pls add a brief meaning, otherwise const is not useful.

Reply:
@alexvanin, @cthulhu-rider, let's discuss what limit should we use and should it be configurable or not

Reply:
TLDR:
1. `maxConnsPerHost = 1` is too small, let's use `maxConnsPerHost = 10`.
2. For NeoFS installations in controlled environment, we can improve performance by increasing `maxConnsPerHost`, so maybe it makes sense to make it configurable.

I did some benchmarks with the client, which fetches 1000 blocks with tx application logs info from the testnet side chain (about 2250 requests in total). 
Here are the results:

| MaxConnsPerHost  | Total duration with 100 routines, sec. | Total duration with 10 routines, sec. |
| ------------- | ------------- | ------------- |
| 0|4.089|20.369
1|120.244|111.339
3|56.372|53.042 
5|37.894|36.364 
10|17.215|24.188
15|9.829|19.737 
20|7.789|20.07

Two things:
- with `MaxConnsPerHost = 0` number of connections equals the number of workers, that's why there is no effect for 10 and 15 `MaxConnsPerHost` with 10 routines,
- `MaxConnsPerHost = 1` affects performance drastically. 

In the storage nodes I expect from 10 to 100 routines call neo-go client to fetch the data (e.g. up to 100 simultaneous `object.Get` request with different container ID). In the private installation we can configure RPC nodes and balance system load so every Storage node can fire up 20, 50 or even 100 connections to RPC node. 

In public networks, I am afraid we can't predict system load and RPC availability. Therefore it makes sense to limit amount of connections. Let's say RPC node processes about 1000 incoming connections. With `maxConnsPerHost = 10` it will process 100 Storage nodes, which is more than enough. 

/cc @realloc @roman-khimov